java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1844,6 +1848,23 @@ public static ColumnVector concatenate(ColumnVector... columns) {
     return new ColumnVector(concatenate(columnHandles));
   }
 
+  /**
+   * Calculate the quantile of this ColumnVector
+   * @param method   the method used to calculate the quantile
+   * @param quantile the quantile value [0,1]
+   */
```

Comment:
Can we add a `@return` statement to explain what is returned by the quantile?  Is it always a double?  Does it match the input type? etc.